### DISCUSSION TITLE
Portable Version

### DISCUSSION THREAD

**Discussion by OxeStone** [TriliumNext/Trilium#3166] [General]
Hello - The portable version creates a user folder 'C:\Users\XXX\AppData\Roaming\Trilium Notes'. It saves all the user settings in that directory and not in the directory specified in the system environment variable 'TRILIUM_DATA_DIR'. How can I change this behavior? Thanks!

**Comment by XXXJetfireXXX** [comment]
Are you launching Trillium using `trillium-portable.bat`? That is needed if you want to use it as portable.

**Comment by OxeStone** [comment]
Yes I am.

**Comment by zadam** [comment]
Hi, `C:\Users\XXX\AppData\Roaming\Trilium Notes` is used by electron for some platform stuff, IIRC downloads of spelling dictionaries and such, but not trilium's "app level" settings which should respect the `TRILIUM_DATA_DIR`

**Comment by OxeStone** [comment]
I think the information about the location and dimension of the app window are saved in 'C:\Users\XXX\AppData\Roaming\Trilium Notes', because when I run the .bat file after deleting the 'C:\Users\XXX\AppData\Roaming\Trilium Notes' folder the app opens with the default window size which is a small square in the middle of the display.

**Reply by zadam** [reply]
Yes, that's correct.

**Comment by OxeStone** [comment]
Is this something that can be fixed soon? It would be nice to have a truly portable app. Thank you.

**Reply by zadam** [reply]
TBH I don't consider saving window position in the portable directory to be critical. It doesn't mean I won't ever find a solution, but it's not a priority.
